src/index.html

```diff
@@ -27,16 +27,16 @@ <h3 class="col-9 px-0 ">Recent scores</h3>
         <div class="col-sm-6">
           <div class="row mt-4">
             <!-- use card-header class later-->
-            <h3 class="px-0 mb-3">Add your score</h3>
+            <h2 class="px-0 mb-3">Add your score</h2>
           </div>
-          <div class="row">
+          <div class="row regular shadow">
             <div class="card text-right">
               <form id="game-form" class="my-3 text-right">
                 <!-- <label id="name-label" for="name-input" class=".sr-only"> Enter your name</label> -->
                 <input id="name-input" type="text" class=" my-2 border-1 p-2 name form-control" type="text"  placeholder="Your name" required/>
                 <!-- <label id="score-label" for="score-input" class=".sr-only"> Enter your score</label> -->
                 <input id="score-input"  type="number" class="my-4 border-1 p-2 score form-control" type="text" placeholder="Your score" required/>
```

Comment:
Great job @ometman 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is regarded as a bad practice.

Reply:
Thank you for that important reminder. I will correct it.